tests/command_line/test_cluster_unit_cell.py

```diff
@@ -27,6 +28,77 @@ def test_dials_cluster_unit_cell_command_line(dials_regression, run_in_tmpdir):
     assert os.path.exists("cluster_unit_cell.png")
 
 
+def test_dials_cluster_unit_cell_command_line_output_files(dials_regression, tmp_path):
+    pytest.importorskip("scipy")
+    pytest.importorskip("xfel")
+
+    data_dir = os.path.join(
+        dials_regression, "refinement_test_data", "multi_narrow_wedges"
+    )
+    experiments = glob.glob(os.path.join(data_dir, "data/sweep_*/experiments.json"))
+    reflections = glob.glob(os.path.join(data_dir, "data/sweep_*/indexed.pickle"))
```

Comment:
I think the following should work:
```suggestion
    data_dir = pathlib.Path(dials_regression) / "refinement_test_data" / "multi_narrow_wedges"
    experiments = data_dir.glob("data/sweep_*/experiments.json")
    reflections = data_dir.glob("data/sweep_*/indexed.pickle")
```